infrastructure/maintenance-container/Dockerfile

```diff
@@ -1,17 +1,13 @@
-FROM ubuntu:latest
+FROM ubuntu:21.10

```

Comment:
The version of ubuntu should be the same as our remote (dev, prod) servers. Ubuntu 21 is not an LTS release (since it's odd numbered), so we shouldn't be using anything newer than Ubuntu 20. Could you confirm what version of Ubuntu we are running on the remote servers?

Reply:
Two months ago Grant said it was 18.04.  PHP 8.0 is not available in the default repos for that release so we would have to use a 3rd party repo like the Ondřej Surý PPA.  If we're comfortable with that I can set that up.  Remember though, that we're not actually serving any pages from the maintenance container.  It's just used to run composer and npm.

Reply:
Until we upgrade from 18 to 20 on the remove servers (since @gggrant mentioned 20 is in fact available, but not 21), could we specify v20 here or does it not have PHP 8.0 by default?

Reply:
Unfortunately 21.10 is the first Ubuntu release to have PHP 8.0 in the official repo as far as I can tell.

Reply:
Ok, I guess we will have to live with that as although I have used the third-party PPA that you mentioned on other projects, it's probably best not to do that here.